CONTRIBUTING.md

```diff
@@ -134,6 +134,35 @@ For Jekyll and its dependencies, because we already have the `Gemfile` in the re
 pushd site && bundle install && popd
 ```
 
+### Use Docker for environemnt setup
+
+We provide a Dockerfile, which installs all the needed development tools inside a Docker container.
+
+One example usage of that, is to build it first:
+
+```
+docker build  -t vega-lite .
+```
+
+To build the current project files, we can mount then the current folder into the container by running:
+
+```
+docker run -ti -v `pwd`:/vega-lite -w /vega-lite -u 1000:100 vega-lite  /bin/bash
```

Comment:
Wow, is all of this needed?

Reply:
Good point.
: -ti   -> run in terminal interactive
: -v `pwd`:/vega-lite     -> mount current directory into Docker, without this docker does not see the checkouted files
: -w /vega-lite . -> changing initial Docker working directory
: -u 1000:1000   -> Docker creates files in the mounted directory. This makes sure they get "my" uid
: vega-lite -> name of the image to run
:/bin/bash -> command to run, by default it would be "nodejs"

This seems "a lot". But people working with Docker use these options everyday.
The same things, each time you use Docker.
So just needed to be learned "once".

Parts of this is coming from the Docker security model. This "isolates" by default container from host, and the user needs to decide deliberately where to break the isolation.

Some of them could be made "default", but then we would assume a certain "usage" of the Dockerfile, which is not always given. 

So in some form it is as well an "example invocation".

Reply:
Same for the Dockerfile itself.  It is a "blue print". A lot of users would change it (slightly !) to adapt to their situation.
It gives a "quick start"

Reply:
done